src/styles/index.scss

```diff
@@ -2,2 +1,57 @@
+@import 'vars';
+@import 'mixins';
+
+.page__body {
   margin: 0;
+  height: 100vh;
+
+  @include flex-block;
+}
+
+.calendar {
+  display: flex;
+  flex-flow: row wrap;
+  gap: $day-gap;
+  width: $calendar-width;
+  padding: 10px;
+
+  &__day {
+    @include flex-block;
```

Comment:
Same here, `flex-block` and `text-style` mixins' names do not give a clear understanding of their purpose. Please, consider renaming them to something more descriptive.